modules/apps/blogs/blogs-service/src/main/java/com/liferay/blogs/service/impl/BlogsEntryLocalServiceImpl.java

```diff
@@ -747,6 +767,32 @@ public BlogsEntry fetchEntry(long groupId, String urlTitle) {
 		return blogsEntryPersistence.fetchByG_UT(groupId, urlTitle);
 	}
 
+	@Override
+	public FileEntry getAttachmentFileEntry(long fileEntryId)
+		throws PortalException {
+
+		FileEntry fileEntry = _portletFileRepository.getPortletFileEntry(
+			fileEntryId);
+
+		_validateAttachmentFileEntry(fileEntry);
+
+		return fileEntry;
+	}
+
+	@Override
+	public FileEntry getAttachmentFileEntryByExternalReferenceCode(
+			long groupId, String externalReferenceCode)
+		throws PortalException {
```

Comment:
The order of the arguments is the reverse; see other occurrences of `get.*ByExternalReferenceCode` is `*ServiceImpl.java` or the usage of `getPortletFileEntryByExternalReferenceCode` just below.

Reply:
OK, I will check, thanks

Reply:
Order changed